spring/spring-boot/documentation/asciidoc/topics/config_examples/hotrod-client.properties

```diff
@@ -0,0 +1,2 @@
+# List Infinispan or Data Grid servers by IP address or hostname at port 11222.
+infinispan.client.hotrod.server_list=127.0.0.1:6667
```

Comment:
@karesti is there any reason why this should be port 6667 for SB and not 11222? I ask because we don't need 2 hotrod-client.properties file examples in the docs.

Reply:
no